### PR TITLE
Release notes 2022-06-26

### PR DESCRIPTION
**Pull request by phealy**

### CODE REVIEW

**Review by miwithro — APPROVED** [review]

**Comment by qpetraroia** [comment]
Thank you for the 1.21 deprecation post!

**Review by qpetraroia — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,32 @@
...
+* Kubernetes 1.21 version deprecation will start taking effect from July 31st, 2022. 
+* Konnectivity rollout will continue in May 2022 and is expected to complete by end of June.
+* Update your AKS labels to the recommended substitutions before deprecation after the Kubernetes v1.24 release. See more information on label deprecations and how to update your labels in the [Use labels in an AKS cluster](https://docs.microsoft.com/azure/aks/use-labels) documentation.
+
+### Release notes
+* Features
+  * [Calico Network Policy](https://docs.microsoft.com/azure/aks/use-network-policies#create-an-aks-cluster-for-calico-network-policies) is now supported for Windows Server 2019 and 2022. This new feature allows customers to use network policies with Windows Server on AKS. Customers can also enable and use both Linux and Windows network policies in a single cluster. This feature will be available from Kubernetes 1.20. Please take note of [common issues related to this change in our troubleshooting documentation.](https://docs.microsoft.com/azure/aks/troubleshooting#windows-containers-have-connectivity-issues-after-a-cluster-upgrade-operation)
```
Comment (palma21) [inline CHANGELOG.md:22]: Not supported, it's now GA no?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,32 @@
...
+
+### Release notes
+* Features
+  * [Calico Network Policy](https://docs.microsoft.com/azure/aks/use-network-policies#create-an-aks-cluster-for-calico-network-policies) is now generally available for Windows Server 2019 and 2022. This new feature allows customers to use network policies with Windows Server on AKS. Customers can also enable and use both Linux and Windows network policies in a single cluster. This feature will be available from Kubernetes 1.20. Please take note of [common issues related to this change in our troubleshooting documentation.](https://docs.microsoft.com/azure/aks/troubleshooting#windows-containers-have-connectivity-issues-after-a-cluster-upgrade-operation)
+* Preview Features
+  * [API Server VNet Integration](https://docs.microsoft.com/azure/aks/api-server-vnet-integration) is available in preview.
+* Component Updates
+  * AKS Ubuntu 18.04 image updated to [AKSUbuntu-1804-2022.06.22](vhd-notes/aks-ubuntu/AKSUbuntu-1804/2022.06.22.txt).
+  * AKS Windows 2022 image updated to [20348.768.220615](vhd-notes/AKSWindows/2022/20348.768.220615.txt).
+  * Application Gateway Ingress Controller add-on has been updated to version 1.5.2.
```
Comment (shashankbarsin) [inline CHANGELOG.md:28]: can we add link to the 1.5.2 release? - https://github.com/Azure/application-gateway-kubernetes-ingress/releases/tag/1.5.2

File: CHANGELOG.md (hunk 1 above)
Comment (phealy) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md (hunk 2 above)
Comment (phealy) [reply CHANGELOG.md]: Thanks, meant to do that.

**Review by shashankbarsin — APPROVED** [review]

**Pull request merged**